assignments/function-conversion.js

```diff
@@ -4,22 +4,40 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+const myFunction = ()=> {
+    console.log("Function was invoked!")
+}
+myFunction(); 
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
+const anotherFunction = (param)=> {
+    return param; 
+}
+console.log(anotherFunction("Example"));
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
+const add = (param1, param2)=> {
```

Comment:
since you are only returning one line, you can remove the curly braces and the word 'return'

```const add = (param1, param2)=> param1+param2;```